clients/ui/frontend/src/__tests__/cypress/cypress/tests/mocked/modelCatalog/modelCatalog.cy.ts

```diff
@@ -0,0 +1,99 @@
+describe('Model Catalog Page', () => {
+  beforeEach(() => {
+    // Visit the page before each test
+    cy.visit('/model-catalog');
+  });
+
+  it('should display loading state initially', () => {
+    cy.contains('Loading model catalog...');
+  });
+
+  it('should display model catalog cards when data is loaded', () => {
+    // Wait for the content to be loaded and cards to be rendered
+    cy.contains('Model Catalog', { timeout: 10000 });
+    cy.contains('Discover models that are available for your organization');
+
+    // Check if model cards are rendered
+    cy.get('[data-testid="model-catalog-card"]', { timeout: 10000 })
```

Comment:
could you please move these commands to `modelCatalog.ts`(create one /cypress/pages) file and use it from there - like how we have in midstream?